task-list/app/controllers/tasks_controller.rb

```diff
@@ -23,11 +24,37 @@ def destroy
     redirect_to "/tasks"
   end
 
+  def complete
+    task = Task.find(params[:id])
+    task.completed_date = Time.now
+    task.completed = true
```

Comment:
It seems like this field could somehow be calculated based on whether or not the `completed_date` field is populated or not